src/octopeer-github/test/DatabaseAdaptable/DatabaseAdapterTest.ts

```diff
@@ -74,7 +74,7 @@ describe("A DatabaseAdapter", function() {
 
         adapter.setDebug();
 
-        adapter.log(1, 1, new Date(), 100, EMPTY_CALLBACK, EMPTY_CALLBACK);
+        adapter.log(new ElementID(1), new EventID(1), new Date(), 100, EMPTY_CALLBACK, EMPTY_CALLBACK);
```

Comment:
Please consider instantiating an `ElementID` and an `EventID` before the tests and reference it here (less duplicate code).